openpype/hosts/maya/plugins/publish/extract_look.py

```diff
@@ -22,6 +24,28 @@
 HARDLINK = 2
 
 
+def get_redshift_tool(tool_name):
+    """Path to redshift texture processor.
+
+    On Windows it adds .exe extension if missing from tool argument.
+
+    Args:
+        tool (string): Tool name.
+
+    Returns:
+        str: Full path to redshift texture processor executable.
+    """
+    redshift_os_path = os.environ["REDSHIFT_COREDATAPATH"]
+
+    redshift_tool_path = os.path.join(
+        redshift_os_path,
+        "bin",
+        tool_name
+    )
+
+    return find_executable(redshift_tool_path)
```

Comment:
Should this maybe be a staticmethod on the `MakeRSTexBin` class. @iLLiCiTiT thoughts?

Reply:
I agree, it's used only there.